docs/public/operators-on-minikube.adoc

```diff
@@ -0,0 +1,127 @@
+[source,sh]
+----
+$ kubectl create -f - << EOF
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: etcd
+  namespace: operators
+spec:
+  channel: clusterwide-alpha
+  name: etcd
+  source: operatorhubio-catalog
+  sourceNamespace: olm
+  startingCSV: etcdoperator.v0.9.4-clusterwide
+  installPlanApproval: Automatic
+EOF
+----
+
+Give it a few seconds before checking the availability of the etcd Operator:
+[source,sh]
+----
+$ odo catalog list services
+Operators available in the cluster
+NAME                                CRDs
+etcdoperator.v0.9.4-clusterwide     EtcdCluster, EtcdBackup, EtcdRestore
+----
+
+==== Troubleshooting
+
+If you don't see etcd Operator using above command or by doing `kubectl get csv -n operators`, make sure that pod belonging to the `CatalogSource` named `operatorhubio-catalog` is running:
+[source,sh]
+----
+$ kubectl get po -n olm | grep operatorhubio-catalog
+----
+
+If the state of this pod is `CrashLoopBackOff`, delete it so that Kubernetes will automatically spin up a new pod for the `CatalogSource`:
+
+[source,sh]
+----
+$ kubectl delete po -n olm <name-of-operatorhubio-catalog-pod>
+----
+
+Once the pod for this `CatalogSource` is up, wait a few seconds before trying to find the etcd Operator when you do `odo catalog list services`.
+
+=== Installing the Service Binding Operator
+
+odo uses the link:https://github.com/redhat-developer/service-binding-operator/[Service Binding Operator] to provide `odo link` feature which links an odo component with an Operator backed service. Thus, to be able to use this feature, it's essential that we install the Operator first.
+
+Service Binding Operator is not yet available via the OLM. The team is link:https://github.com/redhat-developer/service-binding-operator/issues/727[working on making it available] through OLM.
+
+To install the Operator, execute the following `kubectl` command:
+[source,sh]
+----
+$ kubectl apply -f https://github.com/redhat-developer/service-binding-operator/releases/download/v0.1.1-364/install-v0.1.1-364.yaml
+----
+
+You should now see a `Deployment` for Service Binding Operator in the namespace where you installed it:
+[source,sh]
+----
+$ kubectl get deploy -n <replace-namespace-value>
+----
+
+==== Troubleshooting
+
+If linking doesn't work after installing the Service Binding Operator as described above, please open an issue on the odo repository describing the issue. On the issue, please provide the logs of the `Pod` created by the `Deployment` of Service Binding Operator. Execute below command in the namespace where you installed Service Binding Operator (note that `Pod` name will be different in your environment than what's shown below):
+[source,sh]
+----
+$ kubectl get pods
+NAME                                        READY   STATUS    RESTARTS   AGE
+service-binding-operator-65745c4bdc-gc6km   1/1     Running   0          34m
+
+$ kubectl logs service-binding-operator-65745c4bdc-gc6km --follow
```

Comment:
the pod hash will keep changing so instead we can check for the deployment of sbo `kubectl rollout status -w deployment/service-binding-operator -n operators`

Reply:
1. I don't think `kubectl rollout` would show the logs of the pod, would it? Reason for asking pod logs is that it would/could help us determine if something's wrong with how SBO is deployed.
1. We're not mentioning to install this in `operators` namespace. So far, things have been tricky when I have tried to install SBO in `operators` ns. For example, before replying to this comment, I tried to install SBO in `operators` ns on a fresh minikube instance. Here's the logs:
    ```
    {"level":"info","ts":1605702505.1782641,"logger":"controller-runtime.controller","msg":"Starting EventSource","controller":"servicebindingrequest-controller","source":"kind source: operators.coreos.com/v1alpha1, Kind=ClusterServiceVersion"}
    {"level":"error","ts":1605702505.1828096,"logger":"mapper","msg":"listing SBRs","Object.Namespace":"operators","Object.Name":"service-binding-operator-lock","error":"servicebindingrequests.apps.openshift.io is forbidden: User \"system:serviceaccount:operators:service-binding-operator\" cannot list resource \"servicebindingrequests\" in API group \"apps.openshift.io\" at the cluster scope","stacktrace":"github.com/go-logr/zapr.(*zapLogger).Error\n\t/go/src/github.com/redhat-developer/service-binding-operator/vendor/github.com/go-logr/zapr/zapr.go:128\ngithub.com/redhat-developer/service-binding-operator/pkg/log.(*Log).Error\n\t/go/src/github.com/redhat-developer/service-binding-operator/pkg/log/log.go:35\ngithub.com/redhat-developer/service-binding-operator/pkg/controller/servicebindingrequest.(*sbrRequestMapper).Map\n\t/go/src/github.com/redhat-developer/service-binding-operator/pkg/controller/servicebindingrequest/mapper.go:143\nsigs.k8s.io/controller-runtime/pkg/handler.(*EnqueueRequestsFromMapFunc).mapAndEnqueue\n\t/go/src/github.com/redhat-developer/service-binding-operator/vendor/sigs.k8s.io/controller-runtime/pkg/handler/enqueue_mapped.go:67\nsigs.k8s.io/controller-runtime/pkg/handler.(*EnqueueRequestsFromMapFunc).Create\n\t/go/src/github.com/redhat-developer/service-binding-operator/vendor/sigs.k8s.io/controller-runtime/pkg/handler/enqueue_mapped.go:47\nsigs.k8s.io/controller-runtime/pkg/source/internal.EventHandler.OnAdd\n\t/go/src/github.com/redhat-developer/service-binding-operator/vendor/sigs.k8s.io/controller-runtime/pkg/source/internal/eventsource.go:74\nk8s.io/client-go/tools/cache.(*processorListener).run.func1.1\n\t/go/src/github.com/redhat-developer/service-binding-operator/vendor/k8s.io/client-go/tools/cache/shared_informer.go:658\nk8s.io/apimachinery/pkg/util/wait.ExponentialBackoff\n\t/go/src/github.com/redhat-developer/service-binding-operator/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:292\nk8s.io/client-go/tools/cache.(*processorListener).run.func1\n\t/go/src/github.com/redhat-developer/service-binding-operator/vendor/k8s.io/client-go/tools/cache/shared_informer.go:652\nk8s.io/apimachinery/pkg/util/wait.JitterUntil.func1\n\t/go/src/github.com/redhat-developer/service-binding-operator/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:152\nk8s.io/apimachinery/pkg/util/wait.JitterUntil\n\t/go/src/github.com/redhat-developer/service-binding-operator/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:153\nk8s.io/apimachinery/pkg/util/wait.Until\n\t/go/src/github.com/redhat-developer/service-binding-operator/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:88\nk8s.io/client-go/tools/cache.(*processorListener).run\n\t/go/src/github.com/redhat-developer/service-binding-operator/vendor/k8s.io/client-go/tools/cache/shared_informer.go:650\nk8s.io/apimachinery/pkg/util/wait.(*Group).Start.func1\n\t/go/src/github.com/redhat-developer/service-binding-operator/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:71"}
    {"level":"info","ts":1605702505.2789488,"logger":"controller-runtime.controller","msg":"Starting Controller","controller":"servicebindingrequest-controller"}
    {"level":"info","ts":1605702505.2789927,"logger":"controller-runtime.controller","msg":"Starting workers","controller":"servicebindingrequest-controller","worker count":1}
    ```
1. I have mentioned in the doc that pod name will be different from what's shown in the example output.

Reply:
aha , missed `note that `Pod` name will be different in your environment than what's shown below` In that case resolving the convo